src/test_binheap.py

```diff
@@ -193,12 +211,26 @@ def test_bheap_pop_once_right(build_heap_of_five):
 
 def test_bheap_pop_mult(build_heap_of_five):
     """Test pop on multiple values."""
-    build_heap_of_five.pop()
-    build_heap_of_five.pop()
-    build_heap_of_five.pop()
+    assert build_heap_of_five.pop() == 2
+    assert build_heap_of_five.pop() == 6.5
+    assert build_heap_of_five.pop() == 7
     assert build_heap_of_five._list == [15, 27]
 
 
+def test_bheap_pop_all_for_order(build_heap_of_ten):
```

Comment:
I'm still seeing a weird error with your heap when I run it in ipython.

```
b = BinaryHeap([2,6,13,11,16,17,24,29])

In [12]: b.pop()
Out[12]: 2

In [13]: b.pop()
Out[13]: 6

In [14]: b.pop()
Out[14]: 11

In [15]: b.pop()
Out[15]: 17

In [16]: b.pop()
Out[16]: 16

In [17]: b.pop()
Out[17]: 13
```
Pop should return nodes in order from smallest to largest.